### PR TITLE
Initially render examples on the server

### PR DESCRIPTION
**Pull request by taion**
Got a little carried away here, maybe.

This does the initial render of the examples on the server side. It's not perfect - you still need to wait a while for all the JS to finish running on the client before you can interact with any of the examples, but at least you can see them on initial page load.

### CODE REVIEW

**Comment by jquense** [comment]
so excited to try this out

**Comment by taion** [comment]
Oops bug, updating.

**Comment by taion** [comment]
The initial implementation was having issues getting the example to always update. This appears to fix them.

File: docs/src/ReactPlayground.js [hunk 1]
```diff
@@ -1,113 +1,63 @@
...
-const TabPane = modTabPane.default;
-const Thumbnail = modThumbnail.default;
-const Tooltip = modTooltip.default;
-const Well = modWell.default;
-const Overlay = modOverlay.default;
+import babel from 'babel-core/browser';
+import CodeExample from './CodeExample';
 
-/* eslint-enable */
+import SafeAnchor from '../../src/SafeAnchor';
```
Comment (AlexKVal) [inline docs/src/ReactPlayground.js]: `import` => `require` too.
This is also a component.

Comment (AlexKVal) [inline docs/src/ReactPlayground.js]: Ah.. I see.. this is not necessary here.

**Comment by AlexKVal** [comment]
Nice :+1:

**Comment by taion** [comment]
Squashed and added a comment on the `SafeAnchor` import.

You guys okay with my merging this?

**Pull request merged**